README.md

```diff
@@ -253,7 +253,8 @@ A curated list of awesome command-line frameworks, toolkits, guides and gizmos.
 * [bashhub-server](https://github.com/nicksherron/bashhub-server) - Private cloud shell history. Open source server for bashhub
 * [bitpocket](https://github.com/sickill/bitpocket) - "DIY Dropbox" or "2-way directory (r)sync with proper deletion"
 * [Dropbox-Uploader](https://github.com/andreafabrizi/Dropbox-Uploader) - Dropbox Uploader is a Bash script which can be used to upload, download, list or delete files from Dropbox
-* [httpie](https://github.com/httpie/httpie) - HTTPie is a command line HTTP client, a user-friendly cURL replacement
+* [google-drive-upload](https://github.com/labbots/google-drive-upload) - Google drive uploader is a Bash script to upload/sync files to google drive
+* [httpie](https://github.com/jakubroztocil/httpie) - HTTPie is a command line HTTP client, a user-friendly cURL replacement
```

Comment:
Can you revert this change?

Reply:
The issue with URL for httpie is resolved now.